cloud_platform/songs.py

```diff
@@ -1,7 +1,3 @@
 
-def install_exoscale(ctx):
-    ctx.env['cloud.platform'].install('exoscale')
-
-
-def install_ovh(ctx):
-    ctx.env['cloud.platform'].install('ovh')
+def install(ctx):
+    ctx.env['cloud.platform'].install()
```

Comment:
This means that we will need to add manually the installation of a related platform module?